src/AndcultureCode.CSharp.Sitefinity.Core/Extensions/DynamicContentExtensions.cs

```diff
@@ -82,6 +89,29 @@ public static T MapTo<T>(this DynamicContent content, string includes = null) wh
             return mappedItem;
         }
 
+        public static void SetPropertyInDataItem<T>(
+            this T item,
+            SitefinityMetadataAttribute metadataAttribute,
+            ref DynamicContent dataItem,
+            PropertyInfo property
+        )
+        {
+            if (!property.PropertyType.IsSubclassOf(typeof(SitefinityContent)) &&
```

Comment:
Minor: For readability, suggest persisting the result of all of these `IsSubclassOf(...)` as variables at the top of this method.